src/material-experimental/mdc-chips/chip-input.ts

```diff
@@ -156,7 +156,9 @@ export class MatChipInput implements MatChipTextControl, AfterContentInit, OnCha
       // We focus the last chip on backspace only after the user has released the backspace button,
       // And the input is empty (see behaviour in _keyup)
       if (event.keyCode === BACKSPACE && this._focusLastChipOnBackspace) {
-        this._chipGrid._keyManager.setLastCellActive();
+        if (this._chipGrid._chips.length) {
+          this._chipGrid._keyManager.setLastCellActive();
```

Comment:
Shouldn't this be handled by the `GridKeyManager`?
Doesn't it have an indication of whether there are items or not?

Reply:
Yeah, it probably should, but we might want to do it as a follow-up PR. This one has already been tested and confirmed to not break anything (other than the one test that's being investigated). I'd like to avoid broadening the scope and potentially causing more failures

Reply:
NP. Let me know if you'd want me to take care of this later.